vite.config.js

```diff
@@ -7,10 +7,10 @@ export default defineConfig({
     outDir: join(__dirname, "inst/www/react-tools/"),
     lib: {
       // Could also be a dictionary or array of multiple entry points
-      entry: resolve(__dirname, '/srcjs/react-tools.js'),
+      entry: resolve(__dirname, 'srcjs/react-tools.js'),
       name: 'reactR',
-      // the proper extensions will be added
-      fileName: 'react-tools',
+      fileName: () => 'react-tools.js',
+      formats: ['umd'],
```

Comment:
I think the Vite docs are kind of lacking here because it took me a long time to figure this out.. 😆 
https://vitejs.dev/config/build-options#build-lib

But the default formats are `es` and `umd` - we only care about `umd`. And to customize the file extension, you pass a function.

Reply:
Agree that only `umd` is not very well documented at all.  Thanks for finding the secret combination.